pkgs/applications/window-managers/i3/workstyle.nix

```diff
@@ -10,11 +10,11 @@ rustPlatform.buildRustPackage rec {
   src = fetchFromGitHub {
```

Comment:
Please change the version number to ``unstable-2021-05-09``.

Reply:
Of course, that makes more sense.